src/renderer/views/Channel/Channel.js

```diff
@@ -364,7 +367,9 @@ export default Vue.extend({
         } else {
           this.subCount = response.subCount
         }
-        this.thumbnailUrl = response.authorThumbnails[3].url.replace('https://yt3.ggpht.com', `${this.currentInvidiousInstance}/ggpht/`)
+        const thumbnail = response.authorThumbnails[3].url
+        this.thumbnailUrl = thumbnail.replace('https://yt3.ggpht.com', `${this.currentInvidiousInstance}/ggpht/`)
+        this.updateChannelThumbnail(thumbnail, this.channelName, this.id)
```

Comment:
This part worked with
```suggestion
        this.updateChannelThumbnail({
          channelThumbnailUrl: this.thumbnailUrl,
          channelName: this.channelName,
          channelId: this.id
        })
```

Reply:
Did not realize that. I'll make that change when I get the chance. Thanks.

Reply:
I think I did it like this so it saves the youtube thumbnail url and not the invidious one. Will update it to use the named parameters though